packages/react/src/HeroImage/HeroImage.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ImgHTMLAttributes } from 'react'
+import { forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+export interface HeroImageProps extends PropsWithChildren<HTMLAttributes<HTMLElement>> {
+  as?: 'article' | 'aside' | 'div' | 'footer' | 'section'
+  src: ImgHTMLAttributes<HTMLImageElement>['src']
+}
+
+export const HeroImage = forwardRef<HTMLDivElement, HeroImageProps>(
+  ({ as = 'div', children, className, src, ...restProps }: HeroImageProps, ref) => {
+    const Component = as
+
+    return (
+      <Component
+        {...restProps}
+        ref={ref}
+        className={clsx('amsterdam-hero-image', className)}
+        style={{ backgroundImage: `url(${src ?? 'https://picsum.photos/1280/360'})` }}
```

Comment:
Why did you choose a css background image instead of an image (with aspect ratio element) to allow for some a11y. You could use stacking to put the form over an image element:
```
grid-area: 1 / 1;
align-items: center;
```

Reply:
https://codepen.io/dlnr/pen/eYxoaWN

Reply:
A css background will not be described to a screenreader. I think it would be a fair to say something like "image of two people in front of city hall" or something like that.